aws/scenarios/microVMs/microvms/libvirt.go

```diff
@@ -290,7 +290,9 @@ func setupLibvirtVMWithRecipe(instances map[string]*Instance, vmsets []vmconfig.
 		if err != nil {
 			return []pulumi.Resource{}, err
 		}
-		newDomainDepends = append(waitForDomainMatrices, waitKernelHeaders...)
+		newDomainDepends = append(
+			newDomainDepends,
+			append(waitForDomainMatrices, waitKernelHeaders...)...)
```

Comment:
Does `waitForDomainMatrices` need to contain `waitKernelHeaders` for other purposes? If so, you need this on a separate line and capture the return value to be safe.
```
waitForDomainMatrices = append(waitForDomainMatrices, waitKernelHeaders...)
newDomainDepends = append(newDomainDepends, waitForDomainMatrices...)
```

Reply:
I just wanted them concatenated in a single array.
But on second thought this should not be done here anyway. It should be in the provisioning step, so the downloading can be skipped if desired.
I will move it.